Client/Utils/HistoryDeletionUtility.swift

```diff
@@ -112,6 +112,12 @@ class HistoryDeletionUtility: HistoryDeletionProtocol {
         profile.places.deleteHistoryMetadata(since: dateInMilliseconds) { _ in }
     }
 
+    func deleteHistoryMetadataOlderThan(_ dateOption: HistoryDeletionUtilityDateOptions) {
```

Comment:
Could be possible to add unit test for this new function in `HistoryDeletionUtilityTests`?

Reply:
Hey, @yoanarios, I updated the PR with unit test for this function applicable in all scenarios depending on timeframe. It should be good to go now. Thank you so much!